jobs/garden/templates/garden_ctl.erb

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash -e
 
+<% if_p('env.http_proxy') do |http_proxy| %>
+	export HTTP_PROXY=<%= http_proxy %>
+	export http_proxy=<%= http_proxy %>
+<% end %>
+<% if_p('env.https_proxy') do |https_proxy| %>
+	export HTTPS_PROXY=<%= https_proxy %>
+	export https_proxy=<%= https_proxy %>
+<% end %>
+<% if_p('env.no_proxy') do |no_proxy| %>
+	export NO_PROXY=<%= no_proxy %>
+	export no_proxy=<%= no_proxy %>
+<% end %>
```

Comment:
Both variants are likely needed as code is pretty inconsistent about it.  I
would strongly suggest setting both forms - or, you are going to eventually
see a PR that submits for the form that you do not include here.  =)
On Aug 28, 2015 6:07 AM, "Will Pragnell" <notifications@github.com> wrote:

> In jobs/garden/templates/garden_ctl.erb
> <https://github.com/cloudfoundry-incubator/garden-linux-release/pull/5#discussion_r38186560>
> :
>
> > @@ -1,5 +1,18 @@
> >  #!/bin/bash -e
> >
> > +<% if_p('env.http_proxy') do |http_proxy| %>
> > +	export HTTP_PROXY=<%= http_proxy %>
> > +	export http_proxy=<%= http_proxy %>
> > +<% end %>
> > +<% if_p('env.https_proxy') do |https_proxy| %>
> > +	export HTTPS_PROXY=<%= https_proxy %>
> > +	export https_proxy=<%= https_proxy %>
> > +<% end %>
> > +<% if_p('env.no_proxy') do |no_proxy| %>
> > +	export NO_PROXY=<%= no_proxy %>
> > +	export no_proxy=<%= no_proxy %>
> > +<% end %>
>
> I actually think the uppercase might be the more commonly used, no? Given
> that, maybe removing the lowercase would be better?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cloudfoundry-incubator/garden-linux-release/pull/5/files#r38186560>
> .
>